Mention the removal of 4.9.0 and the API breaking changes of upcoming v5 (#170)
DataDog/dogstatsd-ruby: CHANGELOG.md

## 4.9.0 / 2021.03.23

> A version 4.9.0 containing changes intended for 5.0.0 (with API breaking changes) has been released and was available during the 2021-03-23.

Reviewer: ```suggestion
A version 4.9.0 containing changes intended for 5.0.0 (with API breaking changes) has been released and was available on 2021-03-23.
```

> It has been removed on the 2021-03-24 and is not available anymore: v4.8.x should be used for latest v4 version of the gem, upcoming v5.x.x version should be used to benefit of the latest performances improvements.

Reviewer: ```suggestion
It has been removed on 2021-03-24 and is not available anymore: v4.8.x should be used for latest v4 version of the gem, and v5.x.x versions should be used to benefit from the latest performances improvements.
```
